src/cljs/commiteth/core.cljs

```diff
@@ -218,7 +218,7 @@
               [:h3.top-hunters-header "Top 5 hunters"]
               [:div.top-hunters-subheader "All time"]
               [top-hunters]]])]]
-        [footer]]])))
+        ][footer]])))
```

Comment:
Is this intentional or accidental? If intentional the paren should be moved up a line before or after merging.

Reply:
Yes, it's intentional. Shall I edit (push the parent above) and commit the changes here ??

Reply:
fixed it now before the merge. Thanks for pointing it out.